### PR TITLE
Shortcut for manual entry

### LINKED ISSUE #2131: Shortcut for manual entry
- [x] When Timer mode is active (manual mode is off), allow long press on a start button to start a new entry in Manual mode.
- [x] When Manual mode is active (timer mode is off), allow long press on a start button to start a new entry in Timer mode.

### PR DESCRIPTION
**Pull request by willsb**
Closes #2131 

Did on Android first because there already is a `LongClick` binding. When tackling the iOS task (#2030) please check the MvvmCross sources/docs.

Squash whenever, but not before a proper functional test

### CODE REVIEW

File: Toggl.Foundation.Tests/MvvmCross/ViewModels/MainViewModelTests.cs [hunk 1]
```diff
@@ -103,97 +104,82 @@ public void InitializesTheIsInManualModePropertyAccordingToUsersPreferences(bool
...
+        public abstract class BaseStartTimeEntryTest : MainViewModelTest
         {
-            public TheStartTimeEntryCommand()
+            private readonly bool flipManualModeChecks;
+
+            protected abstract ThreadingTask CallCommand();
+
+            protected BaseStartTimeEntryTest(bool flipManualModeChecks)
             {
+                this.flipManualModeChecks = flipManualModeChecks;
```
Comment (daividssilverio) [inline Toggl.Foundation.Tests/MvvmCross/ViewModels/MainViewModelTests.cs]: Clever. To be honest I didn't understand what was going on the tests at a glance. But it made sense when I saw the TheStartTimeEntryCommand and TheAlternativeStartTimeEntryCommand classes and read the ^ documentation.

**Comment by daividssilverio** [comment]
Other than my comments, I tested it on my Nexus and it's working nicely.

File: Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs [hunk 2]
```diff
@@ -261,6 +264,16 @@ private Task startTimeEntry()
...
         }
 
+        private Task alternativeStartTimeEntry()
+        {
+            OnboardingStorage.StartButtonWasTapped();
+
+            var parameter = IsInManualMode
+                ? StartTimeEntryParameters.ForTimerMode(timeService.CurrentDateTime)
+                : StartTimeEntryParameters.ForManualMode(timeService.CurrentDateTime);
+            return navigationService.Navigate<StartTimeEntryViewModel, StartTimeEntryParameters>(parameter);
```
Comment (simonrozsival) [inline Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs]: This whole method is just a copy-paste of the `startTimeEntry` function - we should move this code to a separate `private Task startTimeEntry(bool inManualMode);`.

Comment (willsb) [reply Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs]: True dat

**Pull request merged**